.rspec

```diff
@@ -0,0 +1,6 @@
+--require spec_helper
+--require rspec/instafail
```

Comment:
Why do we need instafail?

Reply:
We need it to check the tests results while creating and fixing them. And that's easier with instafail, because if the test failed for the wrong reason, you can start fixing it while the remaining ones are still running.

Reply:
So it does not breaks the execution on the first fail? 🤔

Reply:
opps, I was thinking about `--fail-fast`